components/brave_rewards/browser/rewards_service.h

```diff
@@ -71,6 +74,10 @@ using GetRewardsMainEnabledCallback = base::Callback<void(bool)>;
 using ConfirmationsHistoryCallback = base::Callback<void(int, double)>;
 using GetRewardsInternalsInfoCallback = base::OnceCallback<void(
     std::unique_ptr<brave_rewards::RewardsInternalsInfo>)>;
+using GetContributionIsInProgressCallback =
+  base::Callback<void(bool)>;
+using OnRemoveRewardsDataCallback = base::Callback<void(bool)>;
+using OnGetContributionIsInProgressCallback = base::Callback<void(bool)>;
```

Comment:
see comment below for `OnGetContributionIsInProgress` - this should be in rewards_service_impl

Reply:
Removed